charts/sn-platform/templates/broker/_broker.tpl

```diff
@@ -36,12 +36,8 @@ Define the kop service url
 Define the web service url
 */}}
 {{- define "pulsar.web.service.url" -}}
-{{- if and .Values.tls.enabled .Values.tls.broker.enabled -}}
```

Comment:
Is it possible for us to support both scenarios, broker TLS for internal cluster addresses and broker TLS for external addresses?

Reply:
Different from Pulsar protocol listener, we can only define one PLAINTEXT port and one TLS port for HTTP protocol, so the tlsWebServicePort will always need to serve traffic with Istio Gateway, the cluster DNS name and Gateway DNS name are always different.